api/Controllers/AuthController.cs

```diff
@@ -35,12 +35,12 @@ public IActionResult Login([FromBody] Login request)
             AccountEntity account = GetAccountByEmail(request.Email);
             if (account == null)
             {
-                return Unauthorized();
+                return BadRequest("Such account does not exist");
             }
             if (!_hashPasswordsService
                 .CheckIfPasswordIsCorrect(account.Password, request.Password, Convert.FromBase64String(account.Salt)))
             {
-                return Unauthorized();
+                return BadRequest("Wrong password");
```

Comment:
Do you see any potential problems with returning different error descriptions in this particular case?